src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -10,7 +10,10 @@ node.name: "{{ elasticsearch_node_name }}"
 {% if elasticsearch_node_zone is defined %}
```

Comment:
We need to make sure that we use different cluster name for ES7 hosts from existing ES2 hosts, so that ES2 and ES7 don't try to join each other (just to be safe)

Reply:
Also, we need to add ES7 datadog tags for ES7 hosts.

Reply:
> Also, we need to add ES7 datadog tags for ES7 hosts.

Added

Reply:
> We need to make sure that we use different cluster name for ES7 hosts from existing ES2 hosts, so that ES2 and ES7 don't try to join each other (just to be safe)

I can't think of how to make it work in role itself. i guess it's upto the user to be cautious.

Reply:
The cluster name can be set in the es config file. I thought we already have  var name for that, don't we? If not, we could add one and make sure to suffix es7 for ES7 hosts?

Reply:
Added an assert statement.